src/ProcessSlave.php

```diff
@@ -425,7 +425,29 @@ protected function handleRequest(ServerRequestInterface $request)
         }
 
         if ($bridge = $this->getBridge()) {
-            $response = $bridge->handle($request);
+            try {
+                $response = $bridge->handle($request);
+            } catch (\Throwable $t) {
+                // PHP >= 7.0
+                error_log(
+                    'An exception was thrown by the bridge. Force restart of the worker. The exception was: ' .
```

Comment:
Force - Forcing

Reply:
Hey I just copied the message from the headers_sent warning a few lines ago :P 
I fixed that message too now :)